README.md

```diff
@@ -126,8 +126,26 @@ Initialize the first node:
 $ swarmd -d /tmp/node-1 --listen-control-api /tmp/manager1/swarm.sock --hostname node-1
```

Comment:
Unrelated to what you are changing, but let's change the path of the socket to be inside /tmp/node-1 instead of /tmp/manager1 (and then update the `export SWARM_SOCKET=...` lines below in the document).